packages/e2e-test-app-fabric/test/LegacyTextInputTest.test.ts

```diff
@@ -0,0 +1,171 @@
+    expect(await textInput.getText()).toBe('abc');
+    await assertLogContains('onKeyPress key: c');
+  });
+  test('Type def on TextInput', async () => {
+    const textInput = await textInputField();
+    await app.waitUntil(
+      async () => {
+        await textInput.setValue('def');
+        return (await textInput.getText()) === 'def';
+      },
+      {
+        interval: 1500,
+        timeout: 5000,
+        timeoutMsg: `Unable to enter correct text.`,
+      },
+    );
+    expect(await textInput.getText()).toBe('def');
+  });
+  /*
+  test('Type hello world on autoCap TextInput', async () => {
+    const textInput = await autoCapsTextInputField();
+    await textInput.setValue('def');
+    expect(await textInput.getText()).toBe('DEF');
+
+    await textInput.setValue('hello world');
+    expect(await textInput.getText()).toBe('HELLO WORLD');
+  });
+  */
+  test('Type abc on multiline TextInput then press Enter key', async () => {
+    const textInput = await textInputField();
+    await textInput.setValue('abc');
+    await textInput.addValue('Enter');
+
+    await assertLogContains('onSubmitEditing text: abc');
+  });
+  test('Type abc on multiline TextInput', async () => {
+    const textInput = await multiLineTextInputField();
+    await textInput.setValue('abc');
+
+    expect(await textInput.getText()).toBe('abc');
+  });
+
+  test('Enter key then type def on multiline TextInput', async () => {
+    const textInput = await multiLineTextInputField();
+
+    await textInput.addValue('End');
+    await textInput.addValue('Enter');
+    await textInput.addValue('def');
+
+    expect(await textInput.getText()).toBe('abc\rdef');
+  });
+
+  test('TextInput onChange before onSelectionChange', async () => {
+    const textInput = await textInputField();
+    await textInput.setValue('a');
+    await assertLogContains('onChange text: a\nonSelectionChange range: 1,1');
+    /*
+    await assertLogContainsInOrder([
+      'onChange text: a',
+      'onSelectionChange range: 1,1',
+    ]);
+    */
+  });
+});
+
+async function textInputField() {
+  const component = await app.findElementByTestID('textinput-field');
+  await component.waitForDisplayed({timeout: 5000});
+  return component;
+}
+
+/*
+async function autoCapsTextInputField() {
+  const component = await app.findElementByTestID('auto-caps-textinput-field');
+  await component.waitForDisplayed({timeout: 5000});
+  return component;
+}
+*/
+
+async function multiLineTextInputField() {
+  const component = await app.findElementByTestID('multi-line-textinput-field');
+  await component.waitForDisplayed({timeout: 5000});
+  return component;
+}
+
+async function assertLogContains(text: string) {
+  const textLogComponent = await app.findElementByTestID('textinput-log');
+  await textLogComponent.waitForDisplayed({timeout: 5000});
+
+  const dump = await dumpVisualTree('textinput-log');
+  expect(dump).toMatchSnapshot();
+  /*
+  await app.waitUntil(
```

Comment:
Left the original LegacyTextInput tests as comments for if we wanted to enable getting the Text out of a Text component, it would be an easy switch.

Reply:
Can you open a tracking issue for enabling the tests later?